Trading/Mode/staggered_orders_trading_mode/staggered_orders_trading_mode.py

```diff
@@ -296,7 +304,9 @@ def _schedule_order_refresh(self):
 
     async def _ensure_staggered_orders_and_reschedule(self):
         await self._ensure_staggered_orders()
-        get_event_loop().call_later(self.HEALTH_CHECK_INTERVAL_SECS, self._schedule_order_refresh)
+        if not self.should_stop:
+            self.scheduled_health_check = get_event_loop().call_later(self.HEALTH_CHECK_INTERVAL_SECS,
```

Comment:
Good idea :+1:

Reply:
But it won't work in backtesting :(

Reply:
It's not required in backtesting, orders can't disapear

Reply:
ok :+1: